src/components/icon/_macro.spec.js

```diff
@@ -91,9 +91,9 @@ describe('macro: icon', () => {
   });
 
   describe.each([
-    ['ons-logo-en', 'Office for National Statistics logo'],
+    ['ons-logo-en', 'Office for National Statistics logo which serves as a hyperlink to the homepage'],
     ['ons-logo-cy', 'Logo Swyddfa Ystadegau Gwladol'],
```

Comment:
Welsh translation needed might be "Hafan Swyddfa Ystadegau Gwladol" but you'll need to check that. This will need to be updated in other places too

Reply:
done